_posts/2015-09-04-survey-improve-api-docu.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Help us improve our API documentation"
+date: "2015-09-04 10:20:00"
+icon: check-square-o
+tags: API documentation development
+categories: api
+authors: ["Birgit"]
+---
+
+Opinions, views and also tastes differ on how an API documentation should be structured, which information should be provided and in which level of detail.
+Of course, the aim is to provide all important and required information to the developers working with our API to keep things simple and easy to handle.
+In order to achieve this, we'd like to involve the most important target group here: developers!
+
+We prepared a short survey that should give some hints on what you would require and expect from an API documentation.
```

Comment:
Just as proposal: We prepared a short survey to find out more about the expectations and requirements you have in order to easily work with an API documentation.